.tool-versions

```diff
@@ -1,4 +1,4 @@
 nodejs 20.9.0
-gradle 8.4
+gradle 8.5
```

Comment:
Fint om denne holdes i sync med `gradlew`-scriptet også. Ser forøvrig ut som at 8.6  er nyeste versjon: https://docs.gradle.org/current/userguide/upgrading_version_8.html

`gradle wrapper --gradle-version 8.6` burde sørge for å oppdatere scriptene. Er mulig du må kjøre `gradle build` for at jar-filen også skal oppdateres, husker ikke helt..